test/tests/functional/pbs_power_provisioning_cray.py

```diff
@@ -226,15 +226,15 @@ def eoe_check(self, jid, eoe, secs):
         nodes = self.job.get_vnodes(self.job.exec_vnode)
         for vname in nodes:
             self.server.expect(VNODE, {'current_eoe': eoe}, id=vname)
-        self.server.expect(JOB, 'job_state', op=UNSET, id=jid, offset=secs)
+        self.server.expect(JOB, 'queue', op=UNSET, id=jid, offset=secs)
         self.host = qstat[0]['exec_host'].partition('/')[0]
         self.mom_logcheck(".*;Job;%s;Cray: get_usage", jid)
-        self.mom_logcheck("capmc get_node_energy_counter --nids *")
-        self.mom_logcheck(".*;Job;%s;energy usage*", jid)
+        self.mom_logcheck("capmc get_node_energy_counter --nids")
+        self.mom_logcheck(".*;Job;%s;energy usage", jid)
         self.mom_logcheck(".*;Job;%s;Cray: pcap node", jid)
-        self.mom_logcheck("capmc set_power_cap --nids *")
-        self.mom_logcheck(".*;Job;%s;PMI: reset current_eoe.*", jid)
-        self.mom_logcheck(".*;Job;%s;Cray: remove pcap node*", jid)
+        self.mom_logcheck("capmc set_power_cap --nids")
+        self.mom_logcheck(".*;Job;%s;PMI: reset current_eoe", jid)
+        self.mom_logcheck(".*;Job;%s;Cray: remove pcap node", jid)
```

Comment:
No need for the preceding '.*' on the above lines.  log_match() will do substring matches.